examples/flows/confidence-level-joining/events.json

```diff
@@ -298,7 +298,21 @@
       }
     },
     "data": {
-      "placeholder": "The syntax of this event has not yet been defined."
+      "outcome": {
+        "verdict": "PASSED",
+        "outcome": "SUCCESSFUL",
+        "description": "Splendid!"
```

Comment:
The description field is intended to "provide human readers with further information". I think "Splendid!" will not educate the humans to make systems that will fill in the correct information. :)